src/main/java/org/session4/homework/ArrayLists.java

```diff
@@ -0,0 +1,24 @@
+package org.session4.homework;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class ArrayLists {
+
+    public static void playWithArrayLists(){
+        ArrayList<String> listOne = new ArrayList<>();
```

Comment:
We can make use of polymorphism and declare List<String> listOne = new ArrayList<>(), since both ArrayList and LinkedList implement the List interface. This comes as a suggestion from my side, there is nothing wrong with the way you wrote it.

Reply:
Sure. However, instead of List, I would write List<String> to make sure I don't have other objects, primitives in there.